### ISSUE TITLE
FZF_DEFAULT_OPTS doesn't work with terminal vim

### ISSUE THREAD

**Issue by ixil** [junegunn/fzf.vim#855]
- Category
    - [x] Bug
- OS
    - [x] Linux
- Vim
    - [x] Vim

In gVim then it works fine, but in terminal vim I get an error, with any of my usual default opts. 
``` 
FZF_DEFAULT_COMMAND=rg --files --no-ignore --hidden --follow --glob "!.git/*"
FZF_DEFAULT_OPTS="--multi"
```

vim message when calling `:Maps`
```
Error running cat '/tmp/vok2Uta/11'|'fzf' --history '/home/ixil/.local/share/fzf-history/maps' --color=bg+:#303030,bg:#1c1c1c,spinner:#00afaf,hl:#808080,fg:#d0d0d0,header:#808080,info:#00afaf,pointer:#5faf5f,marker:#00afaf,fg+:#d0d0d0,prom
pt:#af87d7,hl+:#afd700 --prompt "Maps (n)> " --ansi --no-hscroll --nth 1,.. --color prompt:12 --height=18 > /tmp/vok2Uta/10
```

**Comment by james-ingold** [comment]
I was having a similar issue running BTags, getting an error running cat. The problem was that my fzf version was outdated. I upgraded to 0.18.0 and it started working again. Not sure if it will help you but just throwing it out there.

**Comment by ixil** [comment]
> I upgraded to 0.18.0 and it started working again. Not sure if it will help you but just throwing it out there.

Unfortunately not for me

**Comment by junegunn** [comment]
```
'fzf' --history '/home/ixil/.local/share/fzf-history/maps' --color=bg+:#303030,bg:#1c1c1c,spinner:#00afaf,hl:#808080,fg:#d0d0d0,header:#808080,info:#00afaf,pointer:#5faf5f,marker:#00afaf,fg+:#d0d0d0,prompt:#af87d7,hl+:#afd700 --prompt "Maps (n)> " --ansi --no-hscroll --nth 1,.. --color prompt:12 --height=18
```

Does the command on the error message work on your terminal?

**Comment by ixil** [comment]
sorry for the slow reply, the command exactly as you gave it not work `zsh:
no matches found: --color=bg+:#303030`... however adding single quotes
around that argument making it
```
'fzf' --history '/home/ixil/.local/share/fzf-history/maps'
'--color=bg+:#303030,bg:#1c1c1c,spinner:#00afaf,hl:#808080,fg:#d0d0d0,header:#808080,info:#00afaf,pointer:#5faf5f,marker:#00afaf,fg+:#d0d0d0,prompt:#af87d7,hl+:#afd700'
--prompt "Maps (n)> " --ansi --no-hscroll --nth 1,.. --color prompt:12
--height=18
```
works fine

**Comment by junegunn** [comment]
@ixil Okay, I can easily fix that. But do you know why quoting is necessary on your shell? The command works fine on my zsh.

**Comment by junegunn** [comment]
Not sure but maybe related: https://github.com/robbyrussell/oh-my-zsh/issues/31

**Comment by ixil** [comment]
Thanks this fixed it, my zsh is (5.7.1) 
```
>>> $- 0369BGJNPRVXZghikls
>>> setopt
alwaystoend
autocd
autopushd
nobeep
completeinword
correct
extendedglob
extendedhistory
noflowcontrol
histexpiredupsfirst
histignorealldups
histignoredups
histignorespace
histverify
nohup
incappendhistory
interactive
interactivecomments
login
longlistjobs
nonotify
nopromptcr
nopromptsp
promptsubst
pushdignoredups
pushdminus
sharehistory
shinstdin
vi
```
